test/example_tests/objective_c_example_test.dart

```diff
@@ -22,9 +19,21 @@ void main() {
     });
 
     test('objective_c', () {
-      final pubspecFile = File('example/objective_c/pubspec.yaml');
-      final pubspecYaml = loadYaml(pubspecFile.readAsStringSync()) as YamlMap;
-      final config = Config.fromYaml(pubspecYaml['ffigen'] as YamlMap);
+      final config = testConfig('''
+${strings.name}: AVFAudio
+${strings.description}: Bindings for AVFAudio.
+${strings.language}: objc
+${strings.output}: 'avf_audio_bindings.dart'
+${strings.excludeAllByDefault}: true
+${strings.objcInterfaces}:
+  ${strings.include}:
+    - 'AVAudioPlayer'
+${strings.headers}:
+  ${strings.entryPoints}:
+    - '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/AVFAudio.framework/Headers/AVAudioPlayer.h'
+${strings.preamble}: |
+  // ignore_for_file: camel_case_types, non_constant_identifier_names, unused_element, unused_field, return_of_invalid_type, void_checks, annotate_overrides, no_leading_underscores_for_local_identifiers, library_private_types_in_public_api
```

Comment:
Nit: prefer ignore_for_file on separate lines and alphabetically ordered.

Reply:
I believe this was copied verbatim from the existing pubspec.yaml. see my previous comment, though, about how I plan to make this easier to synchronize by using a standalone config.yaml

Reply:
Please fix the place where this is copied from while your at it. 🙏 

Great! Thanks @TzviPM !